### PR TITLE
Add digitalOcean flk command

### PR DESCRIPTION
**Pull request by jwygoda**

### CODE REVIEW

**Comment by nrdxp** [comment]
maybe `digitalOcean` should just be `doi` for "Digital Ocean Image". I'm not a big fan of camel case in a command line utility. Other than that, it doesn't look all that intrusive.

File: src/pkgs-lib/shell/flk.sh [hunk 1]
```diff
@@ -71,6 +72,12 @@ case "$1" in
     fi
     ;;
 
+  "digitalOcean")
```
Comment (Pacman99) [inline src/pkgs-lib/shell/flk.sh:75]: Unrelated to this PR specifically, but we might want to consider a more generalized `flk build` in the future to encapsulate all the custom builds.

`flk build [iso|doi|home|system]`

File: tests/fullFlake/modules/customBuilds.nix [hunk 2]
```diff
@@ -26,5 +26,8 @@ in
         }
       ];
     })).config.home-manager.users;
+
+    digitalOcean = (mkBuild (diggaLib.modules.digitalOceanConfig {
```
[the remark this answers is not in the document]
Comment (jwygoda) [reply tests/fullFlake/modules/customBuilds.nix]: https://github.com/divnix/devos/pull/302

File: src/modules.nix [hunk 3]
```diff
@@ -115,5 +115,10 @@
       };
     };
 
+  digitalOceanConfig = {}:
+    { modulesPath, ... }: {
+
+      imports = [ "${modulesPath}/virtualisation/digital-ocean-image.nix" ];
+    };
```
[the remark this answers is not in the document]
Comment (jwygoda) [reply src/modules.nix]: https://github.com/divnix/devos/pull/302

**Comment by jwygoda** [comment]
I've changed command name to `doi`

**Review by Pacman99 — APPROVED** [review]
Thank you!
bors r+

**Comment by Pacman99** [comment]
Also for future reference, outside of quick bugfixes, we do most development on the `develop` branch

**Pull request merged**